ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -219,6 +220,19 @@ def test_9_pwpolicy_mod(self):
                                             krbminpwdlife=50)['result']
         assert_attr_equal(entry, 'krbminpwdlife', '50')
 
+        # Test upper bound
+        entry = api.Command['pwpolicy_mod'](
+            self.group, passwordgracelimit=Int.MAXINT)['result']
+        assert_attr_equal(entry, 'passwordgracelimit', str(Int.MAXINT))
+
+        # Test bad values
+        for value in (Int.MAXINT + 1, -2):
+            try:
+                entry = api.Command['pwpolicy_mod'](
+                    self.group, passwordgracelimit=value)['result']
+            except errors.ValidationError:
+                pass
+
```

Comment:
I believe there should be an `else` branch for when no exception is raised, e.g.
```
else:
    assert False
````
alternatively, you can use 
```
with pytest.raises(errors.ValidationError)
   ... your code
```

Reply:
+1. pytest.raises is much better.